castle/cms/cron/_email_audit.py

```diff
@@ -0,0 +1,71 @@
+from castle.cms import audit
+from castle.cms.utils import ESConnectionFactoryFactory
+from AccessControl.SecurityManagement import newSecurityManager
+from plone import api
+from tendo import singleton
+from Products.CMFCore.utils import getToolByName
+from castle.cms.utils import send_email
+
+import transaction
+
+query = {
+    "query": {
+        "range": {
+            "date": {
+                "gte": "now-1d/d"
+            }
+        }
+    }
+}
+
+
+def run_query():
+    registry = getToolByName(app['Castle'], 'portal_registry') # noqa: F821
```

Comment:
in order to not create a different flake8 error, inline comments need to have exactly 2 spaces before comment